docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
     broker:
         image: silintl/idp-id-broker:develop
         environment:
-            API_ACCESS_KEYS: local-sync-123
+            API_ACCESS_KEYS: local-sync-123,codeship-sync-to-broker-11111111
```

Comment:
I don't think the `codeship-sync-to-broker-11111111` is needed here, since the `docker-compose.yml` file isn't used by Codeship. Instead, Codeship uses the `codeship-services.yml` file.